rune/libcontainer/specconv/spec_linux.go

```diff
@@ -334,53 +334,57 @@ func createEnclaveConfig(spec *specs.Spec, config *configs.Config) {
 		args = strings.Join(a, " ")
 	}
 
-	isProductEnclave := filterOut(env, "ENCLAVE_IS_PRODUCT_ENCLAVE")
-	if isProductEnclave == "" {
-		isProductEnclave = libcontainerUtils.SearchLabels(config.Labels, "enclave.is_product_enclave")
-	}
-	var is_product_enclave uint32
-	if strings.EqualFold(isProductEnclave, "false") {
-		is_product_enclave = sgx.DebugEnclave
-	} else if strings.EqualFold(isProductEnclave, "true") {
-		is_product_enclave = sgx.ProductEnclave
-	} else {
-		is_product_enclave = sgx.InvalidEnclaveType
-	}
-
 	raType := filterOut(env, "ENCLAVE_RA_TYPE")
 	if raType == "" {
 		raType = libcontainerUtils.SearchLabels(config.Labels, "enclave.attestation.ra_type")
 	}
 	var ra_type uint32
-	if strings.EqualFold(raType, "EPID") {
+	if raType == "" {
+		ra_type = sgx.UnsetRaType
```

Comment:
if raType is not specified, you should **NEVER** care about any other settings about RA.

```go
if raType != "" {
  // parse enclave.attestation.* and ENCLAVE_RA_*
}
```